rpcs3/Emu/Cell/lv2/sys_fs.h

```diff
@@ -148,16 +148,20 @@ enum class lv2_file_type
 struct lv2_fs_mount_point
 {
 	const std::string_view root;
+	const std::string_view file_system;
+	const std::string_view device;
 	const u32 sector_size = 512;
 	const u64 sector_count = 256;
 	const u32 block_size = 4096;
 	const bs_t<lv2_mp_flag> flags{};
+	u8 is_mounted = false;
```

Comment:
this is not a boolean

Reply:
I use it to store the mount status of 8 dev_usbXXX devices, for the other devices, it is used as a boolean.

Reply:
Done.